src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: $white-accent-color;
```

Comment:
The variable `$white-accent-color` is used here, but it's not defined in this file. Ensure that `$white-accent-color` is defined in a global SCSS file or imported into this file to avoid compilation errors.